Updated based on review comment (#5889)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

### Step 1 - Configure Choreo Connect

#### Step 1.1 - Compile the reporter implementation

Follow the instructions below to compile the reporter implementation as a JAR file:

1. Checkout and build the [wso2/samples-apim](https://github.com/wso2/samples-apim) repository.

2. Navigate to the `analytics-event-publisher/target` directory.
3. Copy and paste the generated JAR in to the `choreo-connect-1.x.x/docker-compose/resources/enforcer/dropins` directory.

>      This JAR will be mounted in to the `/home/wso2/lib/dropins` directory within the Enforcer when the Enforcer starts.

Reviewer: ```suggestion
     This JAR will be mounted into the `/home/wso2/lib/dropins` directory within the Enforcer when the Enforcer starts.
```
